pkg/selectors/kernel.go

```diff
@@ -536,17 +537,16 @@ func writeMatchAddrsInMap(k *KernelSelectorState, values []string) error {
 	for _, v := range values {
 		addr, maskLen, err := parseAddr(v)
 		if err != nil {
-			return fmt.Errorf("MatchArgs value %s invalid: %w", v, err)
+			return fmt.Errorf("MatchArgs value %s invalid: parse IP: %w", v, err)
 		}
-		if len(addr) == 4 {
+
+		if len(addr) == net.IPv4len {
 			val := KernelLPMTrie4{prefixLen: maskLen, addr: binary.LittleEndian.Uint32(addr)}
 			m4[val] = struct{}{}
-		} else if len(addr) == 16 {
+		} else {
 			val := KernelLPMTrie6{prefixLen: maskLen}
 			copy(val.addr[:], addr)
 			m6[val] = struct{}{}
-		} else {
```

Comment:
How about we keep this else? I know it's not strictly needed, but I feel that it makes the code more readable and provides a sanity check that `parseAddr()` works as expected.

Reply:
Agree, returned this `else`